src/lib/conseil.js

```diff
@@ -0,0 +1,1101 @@
+const conseiljs = require('conseiljs')
```

Comment:
Lots of undefined variables here. Needs to be cleaned up.

Reply:
![image](https://user-images.githubusercontent.com/7572532/114323187-f7722b00-9ad8-11eb-89ea-28e69eafa7b0.png)

Reply:
Getting same results as master when above is fixed.

Reply:
What does this mean? They need to be predefined?